src/main/java/core/basesyntax/AreaCalculator.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
```

Comment:
It would be great if you will make a packages an put figures to 'figure' package, suppliers to 'supplier' etc